src/status_im2/contexts/chat/lightbox/utils.cljs

```diff
@@ -0,0 +1,164 @@
+(ns status-im2.contexts.chat.lightbox.utils
+  (:require
```

Comment:
This file is not a new code. These methods where initially in the `view` file.

Reply:
Yes correct. But feel free to share your comments (tho I am sure you reviewed this code before :) )

Reply:
I did, and you know @OmarBasem, code reviews are funny, it's like coding where the solution usually doesn't come out easily, requires trial and error. Similarly, in code reviews, as I familiarize myself with the new code (PR after PR being merged), the fog gradually dissipates.

Well, enough of comparisons!